### ISSUE TITLE
[docs]: Mobile tap target area for the all topics button should be increased

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#1292]
At the moment the all topics button is as wide as the arrow + the text that it has which is too small less than 23px in height and the width is under 95px.we should probably add some padding around it in the y-direction and probably make it take the full available space so that it is easier to tap on via a mobile device. 

![image](https://user-images.githubusercontent.com/46004116/144137879-7648bba7-6e47-43a4-ad30-2c6faaa0f3cc.png)
